app/models/bus_stop.rb

```diff
@@ -48,8 +48,21 @@ def route_list
     routes.pluck(:number).sort.join(', ')
   end
 
-  def self.to_csv(limited_attributes: false)
-    # TODO
+  def self.to_csv
+    CSV.generate headers: true do |csv|
```

Comment:
I don't think `headers: true` does anything for `generate`.

Reply:
Yeah, it doesn't make any difference to the csv.